packages/pwa/app/commerce-api/auth.js

```diff
@@ -94,29 +94,21 @@ class Auth {
     }
 
     /**
-     * Called with the details from the redirect page that _loginWithCredentials returns
```

Comment:
Across methods, there was a lot of converting camelcase variables names used in this codebase to under scores required by the API.

I've opted to save a few lines by using underscores.